src/tests/ftest/util/agent_utils_params.py

```diff
@@ -32,6 +33,43 @@ def _get_new(self):
         return DaosAgentTransportCredentials(self._log_dir)
 
 
+class DaosAgentTelemetryCredentials(TelemetryCredentials):
+    # pylint: disable=too-few-public-methods
+    """Telemetry credentials listing certificates for secure communication."""
+
+    def __init__(self, log_dir=os.path.join(os.sep, "tmp")):
+        """Initialize a TelemetryConfig object."""
+        super().__init__("/run/agent_config/telemetry_config/*", None, log_dir)
+
+        self.port = BasicParameter(None, 9192)
+        self.enabled = BasicParameter(None)
+        self.retain = BasicParameter(None)
+        self.server_cert = LogParameter(self._log_dir, None, "telemetryserver.crt")
+        self.server_key = LogParameter(self._log_dir, None, "telemetryserver.key")
+
+    def get_certificate_data(self, name_list):
+        """Get certificate data.
+
+        Args:
+            name_list (list): list of certificate attribute names.
+
+        Returns:
+            data (dict): a dictionary of parameter directory name keys and
+                value.
+
+        """
+        data = super().get_certificate_data(name_list)
+        return data
+
```

Comment:
This function just returns the `super()` so it's not necessary
```suggestion
```

Reply:
Will remove it.